src/git-duet/git-duet-rebase/main.go

```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"fmt"
+	"os"
+
+	"git-duet/internal/cmd"
+	"git-duet/internal/cmdrunner"
+)
+
+func main() {
+	err := cmdrunner.Execute(
+		cmd.New("rebase"))
```

Comment:
I'm not seeing the `--signoff` flag present on `git rebase`. Supporting `rebase` may require some more gymnastics unless I'm missing something. See https://gist.github.com/nh2/66e1cd83a86da7fe43b1 for example.